### PR TITLE
update english translations

### PR DESCRIPTION
**Pull request by ruslandoga**
This PR updates English translations and starts using CLDR instead of scraping Wikipedia.

### CODE REVIEW

File: mix_tasks/update_english_translations.ex [hunk 1]
```diff
@@ -1,8 +1,223 @@
...
+  @translations_dest Application.app_dir(:location, "/priv/iso_3166-2.en-translations.json")
+
   def run(_) do
-    Location.Country.load()
-    Location.Scraper.scrape()
+    {200, _headers, cldr_xml} = Location.HTTP.get!(@cldr_url)
+    cldr_translations = parse_cldr_xml(cldr_xml)
+
+    %{"3166-2" => debian} = read_json("priv/iso_3166-2.json")
+    restored = read_json("priv/restore/iso_3166-2.json")
```
Comment (ukutaht) [inline mix_tasks/update_english_translations.ex:12]: What's the purpose of the restore file?

Comment (ruslandoga) [reply mix_tasks/update_english_translations.ex:12]: To remember the ISO codes that no longer exist in the real world but are still kept in Plausible database.

Comment (ruslandoga) [reply mix_tasks/update_english_translations.ex:12]: Relevant PR where it was added: https://github.com/plausible/location/pull/17/files

**Review by ukutaht — APPROVED** [review]

File: mix_tasks/update_english_translations.ex [hunk 2]
```diff
@@ -123,6 +123,8 @@ defmodule Mix.Tasks.UpdateEnglishTranslations do
     "KZ-75" => "Almaty City",
     # https://en.wikipedia.org/wiki/ISO_3166-2:FI
     "FI-01" => "Åland",
+    "FI-06" => "Kanta-Häme",
+    "FI-16" => "Päijät-Häme",
```
Comment (ruslandoga) [inline mix_tasks/update_english_translations.ex:127]: https://chatgpt.com/share/1c672980-68ff-4f92-ba92-869825d61d47

**Pull request merged**